src/main/resources/feedbackQuestionTextSubmissionFormTemplate.html

```diff
@@ -1,9 +1,17 @@
 <textarea rows="4" cols="100%" class="form-control"
     ${disabled} id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
     name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    data-lengthTextId="responseLength-${questionIndex}-${responseIndex}">${existingResponse}</textarea>
-<div class="pull-right margin-top-7px">
-    <label class="text-color-gray font-weight-normal">Response length: </label>
-    <label class="text-color-gray font-weight-normal" id="responseLength-${questionIndex}-${responseIndex}">0</label>
-    <label class="text-color-gray font-weight-normal"> characters</label>
-</div>
+    data-lengthTextId="responseLength-${questionIndex}-${responseIndex}" data-recommendedText="${recommendedLength}">${existingResponse}</textarea>
```

Comment:
I missed this out earlier, it would be preferred if you used hyphens to delimit the data attribute. i.e. `data-length-text-id` and `data-recommended-text`